packages/numbers/lib/numbers.ts

```diff
@@ -82,25 +135,51 @@ export class BaseAPI {
 
 export class Numbers extends BaseAPI {
 
-    public buyNumber(params?: NumbersParams) {
+    public async buyNumber(params?: NumbersParams) {
         const localVetchOptions = NumbersParamCreator(this.config).buyNumber(params);
-        return runRequest<NumbersEmptyResponse>(localVetchOptions);
+        const resp = await runRequest<NumbersEmptyResponse>(localVetchOptions);
+
+        return {
+            errorCode: resp.data['error-code'],
+            errorCodeLabel: resp.data['error-code-label']
+        };
     }
-    public cancelNumber(params?: NumbersParams) {
+    public async cancelNumber(params?: NumbersParams) {
         const localVetchOptions = NumbersParamCreator(this.config).cancelNumber(params);
-        return runRequest<NumbersEmptyResponse>(localVetchOptions);
+        const resp = await runRequest<NumbersEmptyResponse>(localVetchOptions);
+
+        return {
+            errorCode: resp.data['error-code'],
+            errorCodeLabel: resp.data['error-code-label']
+        };
     }
-    public getAvailableNumbers(filter?: NumbersSearchFilter) {
+    public async getAvailableNumbers(filter?: NumbersSearchFilter) {
         const localVetchOptions = NumbersParamCreator(this.config).getAvailableNumbers(filter);
-        return runRequest<NumbersAvailableList>(localVetchOptions);
+        const resp = await runRequest<NumbersAvailableList>(localVetchOptions);
+
+        let data: NumbersAvailableList = {
```

Comment:
```suggestion
        const data: NumbersAvailableList = {
```
I think this could be a const cause you aren't changing it